realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -118,10 +118,19 @@ public boolean shouldDeleteRealmIfMigrationNeeded() {
         return durability;
     }
 
-    public RealmProxyMediator getSchemaMediator() {
+    RealmProxyMediator getSchemaMediator() {
```

Comment:
To follow our normal deprecation policies this should be marked as deprecated first before removing the public modifier.